Makefile

```diff
@@ -633,12 +633,20 @@ fmt: ## Format the code using Ruff and blacken-docs
 	python -m ruff check --fix-only
 	python -m ruff format
 	@echo "Formatting the code blocks with blacken-docs..."
+	# TODO: Update format to use py311 for Python 3.11
```

Comment:
If we have the python version available in MLRUN_PYTHON_VERSION, why not use that to determine which value gets passed to the `-t` argument?